README.md

```diff
@@ -7,7 +7,7 @@
 
     $ git clone git@github.com:OCA/maintainers-tools.git
     $ cd maintainers-tools
-    $ virtualenv env
+    $ python3 -m venv env
```

Comment:
Note that to get something really equivalent you'll need to add `env/bin/pip install -U pip setuptools wheel` to get the latest `pip` and `setuptools` and more importantly to add `wheel`. Indeed python's venv module does not install `wheel` and when it is absent, pip downgrades to legacy behaviour in several code paths.

Reply:
This means that `wheels` installation is also needed?

Reply:
@dreispt it's not strictly necessary but when it's absent, pip falls back to legacy behaviours. So better take good habits.

Reply:
Now that pip does not do `setup.py install` anymore, I'm now comfortable with recommending `python3 -m venv` followed by a pip upgrade.